modules/file-system/parallelstore/scripts/install-daos-client.sh

```diff
@@ -52,7 +52,9 @@ EOF
 		fi
 
 		## TODO: Remove disable automatic update script after issue is fixed.
-		/usr/bin/google_disable_automatic_updates
+		if [ -x usr/bin/google_disable_automatic_updates ]; then
```

Comment:
@wiktorn by eye, there is a missing leading slash

Reply:
Good catch, resolved!